smaps/src/main/webapp/js/memory-map.js

```diff
@@ -212,3 +229,120 @@ function getColor(permissions) {
       return 'White';  // White.
   }
 }
+
+/* Creates the box of information displayed at the right of the screen with the
+ * specified region's fields.
+ */
+function drawRegionFocus(region) {
+  // Get the button object that the region info will go in.
+  const regionFocusContainer =
+      document.getElementById('region-focus-container');
+
+  // Create a button object to put all the region info in.
+  const regionFocusBtn = document.createElement('button');
+
+  // Style the button.
+  regionFocusBtn.className = 'region-focus';
+
+  // Create a json object with key value pairs for the field name and how it
+  // will be displayed to the user.
+  const fieldNames = createFieldNames();
+
+  // Go through every field in the region.
+  for (field in region) {
+    // Print the proper label, the value, and if the field is one that is
+    // represented in kB, add a kB to the end of the string, otherwise don't.
+    let textNode;
```

Comment:
I'm really out of my comfort zone with .js, but...

woule it make sense to

let textNode = document.createTextNode(fieldnames[field] + ": " + region[field])
if (isKB(field)) {
  textNode = ...
}

I.e.,
initialize textNode with the 'else' option and then not have the else?

Reply:
Yes that makes sense I can do that! Makes it a bit clearer.